Kickstarter-iOS/Views/Cells/PledgeCreditCardView.swift

```diff
@@ -46,6 +51,8 @@ final class PledgeCreditCardView: UIView {
     _ = (self.rootStackView, self)
       |> ksr_addSubviewToParent()
       |> ksr_constrainViewToMarginsInParent()
+
+    self.selectButton.addTarget(self, action: #selector(self.selectButtonTapped), for: .touchUpInside)
```

Comment:
This looks like SwiftFormat ... I think we've raised this before that we'd prefer to use class name here `#selector(PledgeCreditCardViewController.selectButtonTapped)` 😉